.github/workflows/ci.yml

```
@@ -0,0 +1,14 @@
+name: Continuous Integration
+on: [push]
```

you can use [pull-request](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#example-using-a-list-of-events)

```suggestion
on: [pull_request]
```